com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/launch/ModelServerEntrypoint.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2019 EclipseSource and others.
+ *
+ *   This program and the accompanying materials are made available under the
+ *   terms of the Eclipse Public License v. 2.0 which is available at
+ *   http://www.eclipse.org/legal/epl-2.0.
+ *
+ *   This Source Code may also be made available under the following Secondary
+ *   Licenses when the conditions for such availability set forth in the Eclipse
+ *   Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ *   with the GNU Classpath Exception which is available at
+ *   https://www.gnu.org/software/classpath/license.html.
+ *
+ *   SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ *******************************************************************************/
+package com.eclipsesource.modelserver.emf.launch;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.eclipsesource.modelserver.common.AppEntrypoint;
+import com.eclipsesource.modelserver.common.Routing;
+import com.google.inject.Inject;
+
+import io.javalin.Javalin;
+import io.javalin.JavalinEvent;
+
+public class ModelServerEntrypoint implements AppEntrypoint {
+	
+	private Javalin app;
+	private static final Logger logger = LoggerFactory.getLogger(ModelServerLauncher.class);
```

Comment:
Maybe we should agree on common naming for the Logger instances.
I used LOG but I don't have a preference, I'd just like it to be consistent.